### PR TITLE
Changes to introduction

### PR DESCRIPTION
**Pull request by dmurvihill**

### CODE REVIEW

**Comment by dmurvihill** [comment]
Hey, can you take a look at this?

**Comment by mheon** [comment]
This is not in keeping with Lauer's requests from earlier (email "Intro and Definitions" from Dec. 23 of last year). He suggests an alternative structure for the introduction that I think we should follow. Please redraft given this.

I will be working on this over the weekend, mostly on the latter sections of the report. I believe we still don't have a conclusion. I will fix this, and the MOSS comparison.

Also I am probably going to make a Github organization for Checksims. And move a copy of the repo into it. I am debating leaving the report out and making it code exclusive (we can continue hacking on the report in our local repos).

**Comment by dmurvihill** [comment]
That sounds like a good way to proceed.

**Comment by dmurvihill** [comment]
Okay, this should address the 12/23 comments.

File: doc/src/intro.ltx [hunk 1]
```diff
@@ -1,43 +1,39 @@
 \section{Introduction}
 \label{sec:intro}
 Over the past year, we have heard reports of a number of cases of academic
```
Comment (mheon) [inline doc/src/intro.ltx]: I don't like this first sentence. Which I wrote, so I shouldn't complain, but still... I feel that there's a better way to open this.

File: doc/src/intro.ltx [hunk 2]
```diff
@@ -1,43 +1,39 @@
...
-dishonesty occurring at Worcester Polytechnic Institute. Specifically, most of
-these have concerned the unauthorized and unacknowledged copying of source code
-within the Electrical and Computer Engineering and Computer Science departments.
-The first reports came from Nicholas DeMarinis, a Teaching Assistant in the
-Electrical and Computer Engineering department, who noticed several instances
-of source code which he considered to be suspiciously similar in an embedded
-systems programming course. He identified several cases of academic dishonesty,
-including one where the students had extensively obfuscated the copied code.
+dishonesty occurring in the Computer Science department and the Electrical and
+Computer Engineering department at Worcester Polytechnic Institute.
```
Comment (mheon) [inline doc/src/intro.ltx]: Departments, plural, we mention two

Comment (mheon) [inline doc/src/intro.ltx]: Nevermind, we have a separate "department" for each... I still feel like departments is better, just drop the first use as well. Computer Science and Electrical and Computer Engineering departments, or somesuch?

File: doc/src/intro.ltx [hunk 3]
```diff
@@ -1,43 +1,39 @@
...
+Most have concerned the unauthorized and unacknowledged
+copying of source code. The first reports came from Nicholas DeMarinis,
```
Comment (mheon) [inline doc/src/intro.ltx]: Feel like we should disambiguate source code, this is the first time we're using it... Copying of source code in programming assignments?

File: doc/src/intro.ltx [hunk 4]
```diff
@@ -1,43 +1,39 @@
...
-Both courses had more than one teaching assistant; if the two assignments which
-contained unauthorized copying were graded by different teaching assistants,
-the copying would almost certainly not have been identified. This draws
-questions to how many cases of academic dishonesty occurred which were not
-identified.
+In both cases, the unauthorized copying was caught mostly by luck.
+Both courses had more than one teaching assistant; if the copied assignments
+had been graded by different teaching assistants, the copying would almost
+certainly not have been identified. The course staff could not have been so
+lucky all the time, so many cases of unauthorized copying have most likely
```
Comment (mheon) [inline doc/src/intro.ltx]: Don't like this "not have been so lucky" - Instead, should say "Statistically, it is unlikely that the same TA has graded all such instances of unauthorized copying" or similar. More professional.

File: doc/src/intro.ltx [hunk 5]
```diff
@@ -1,43 +1,39 @@
...
-to deal with. A solution for flagging potential cases of unauthorized copying
-for review by course staff has the potential to greatly increase detection of
-academic dishonesty.
- 
-Professor Lauer commissioned us, the authors, to construct a system for
-automatically detecting suspicious assignments, with the intent of integrating
-it into his low-level courses as a tool to identify which assignments require
-manual review.
+The copying does not slip through because the course staff cannot recognize it.
+Dishonest students escape because there are so many assignments that the staff
```
Comment (mheon) [inline doc/src/intro.ltx]: The staff does review them all, just not the same staff member - important distinction

File: doc/src/intro.ltx [hunk 6]
```diff
@@ -51,16 +47,16 @@ definitions of academic dishonesty is contained in
...
 In Professor Lauer's lower level courses, students are encouraged to collaborate
-on a whiteboard and design pseudocode solutions, but they must type in and
-debug their own programs from these solutions. The direct copying of source
-code is, however, prohibited. This way, the code must pass ``through the
-brain'' --- the students will gain more understanding by typing
-independently than by copying code directly. Direct copying from other sources
-does constitute academic dishonesty. Such rules are described in the course
-syllabus, and are usually explained directly during the first lecture as well.
+on a white board and design pseudocode solutions, but they must type in and
+debug their own programs from the pseudocode. The direct copying of source code
```
Comment (mheon) [inline doc/src/intro.ltx]: This pseudocode? We mention it in the previous sentence

File: doc/src/intro.ltx [hunk 7]
```diff
@@ -51,16 +47,16 @@ definitions of academic dishonesty is contained in
...
+is prohibited. This way, the code must pass ``through the brain''~--- the
+students will gain more understanding by typing independently than by copying
+code directly. Direct copying does constitute academic dishonesty. Such rules
```
Comment (mheon) [inline doc/src/intro.ltx]: I don't like the jump straight into "Direct copying does constitute..." --- maybe use "Direct copying is considered to be" instead?

File: doc/src/intro.ltx (hunk 2 above)
Comment (dmurvihill) [inline doc/src/intro.ltx]: Did it your way and it reads better.

File: doc/src/intro.ltx (hunk 3 above)
Comment (dmurvihill) [inline doc/src/intro.ltx]: Now says "copying of program source code"

File: doc/src/intro.ltx (hunk 4 above)
Comment (dmurvihill) [inline doc/src/intro.ltx]: Your point is well-taken, but I am also worried about sounding overly stuffy. I think I hit the right balance this time.

File: doc/src/intro.ltx (hunk 5 above)
Comment (dmurvihill) [inline doc/src/intro.ltx]: Fixed.

File: doc/src/intro.ltx (hunk 6 above)
Comment (dmurvihill) [inline doc/src/intro.ltx]: Using "this" to refer to something introduced in the previous sentence is a peeve of mine. I think what I ended up with is better.

File: doc/src/intro.ltx (hunk 7 above)
Comment (dmurvihill) [inline doc/src/intro.ltx]: Added "In Professor Lauer's view"

**Comment by mheon** [comment]
Looks good, pulling

**Pull request merged**